applications/ParticleMechanicsApplication/python_scripts/particle_mechanics_analysis.py

```diff
@@ -48,11 +48,11 @@ def RunSolutionLoop(self):
             self.OutputSolutionStep()
 
             ## Stop solution loop timer
-            end_solve_time = time.time()
+            end_solve_time = time()
             KratosMultiphysics.Logger.PrintInfo(self._GetSimulationName(), "SOLVING TIME: ", end_solve_time - start_solve_time, " s]")
 
         ## Stop analysis timer
-        analysis_end_time = time.time()
+        analysis_end_time = time()
```

Comment:
check out #5232 maybe you don't need your custom timer stuffs

Reply:
Agree with you. #5232 is a very cool feature. Will remove the custom timer!